### PR TITLE
APIDOC-114: Update Sign-in button destination URL

### PR DESCRIPTION
**Pull request by VinayB09**
Update the Sign in button destination URL to https://identity.nexmo.com/login

### CODE REVIEW

File: config/business_info.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ header:
         - get_an_api_key
         - start_building
     sign-in:
-      path: https://dashboard.nexmo.com/sign-in?icid=nexmocustomer_api-developer-adp_nexmodashbdsigin_nav
+      path: https://identity.nexmo.com/login
```
Comment (pardel) [inline config/business_info.yml:15]: @niati @VinayB09 Do we want to remove the tracking information here?

**Review by pardel — CHANGES_REQUESTED** [review]

File: config/business_info.yml (hunk 1 above)
Comment (niati) [reply config/business_info.yml:15]: @pardel you mean from dashboard.nexmo.com? If so, BI told us to add the tracking to the new URL.

Comment (niati) [reply config/business_info.yml:15]: I see what you're saying - it appears that we didn't add it to the identity URL.

Comment (VinayB09) [reply config/business_info.yml]: Added icid to the. new URL.

**Review by marcoranieri — APPROVED** [review]

**Pull request merged**